python/src/aac/plugins/validators/defined_references/_validate_references.py

```diff
@@ -18,25 +18,25 @@ def validate_references(definition_under_test: Definition, target_schema_definit
     Returns:
         A ValidatorResult containing any applicable error messages.
     """
-    error_messages = []
-
-    def validate_dict(dict_to_validate: dict) -> list[str]:
+    findings = ValidatorFindings()
+    *_, validation_name = __package__.split(".")
```

Comment:
This is not the name we actually want - i.e. it will return `defined_references` instead of `Type references exist`. I'm working on refactoring this to give the actual validation name but haven't gotten all the kinks worked out, yet. I'll push an update with this soon.